src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs

```diff
@@ -154,13 +155,17 @@ private bool LineContainsQuote(ITextSnapshotLine line, int caretPosition)
             var useTabs = options.GetOption(FormattingOptions.UseTabs);
             var tabSize = options.GetOption(FormattingOptions.TabSize);
             var indentStyle = options.GetOption(FormattingOptions.SmartIndent, LanguageNames.CSharp);
+            var lineEnding = options.GetOption(FormattingOptions.NewLine, LanguageNames.CSharp);
 
             var root = document.GetSyntaxRootSynchronously(cancellationToken);
             var sourceText = root.SyntaxTree.GetText(cancellationToken);
+            var newLineTrivia = lineEnding == "\n" ? SyntaxFactory.TriviaList(SyntaxFactory.ElasticLineFeed) : SyntaxFactory.TriviaList(SyntaxFactory.ElasticCarriageReturnLineFeed);
```

Comment:
```suggestion
            var newLineTrivia = SyntaxFactory.TriviaList(lineEnding == "\n" ? SyntaxFactory.ElasticLineFeed : SyntaxFactory.ElasticCarriageReturnLineFeed);
```

Reply:
@jasonmalinowski is this the right way for features to determien appropriate line ending?  I seem to recall you had a PR where there was some core service that could be used that would figure this out based both on settings and heuristics about the file in question.  DId that go through?  Should that be used here instead?

Thanks!